app/javascript/components/components/Header.js

```diff
@@ -1,23 +1,41 @@
 import React from "react";
 import { Nav, NavItem, NavLink } from "reactstrap";
 
+
 const Header = (props) => {
+  console.log(props)
+  const {logged_in, current_user} = props
   return (
     <div>
-      <p>List Based</p>
       <Nav>
         <NavItem>
           <NavLink href="/">Home</NavLink>
         </NavItem>
         <NavItem>
           <NavLink href="/AboutUs">About</NavLink>
         </NavItem>
-        <NavItem>
-          <NavLink href="/users/sign_in">Login</NavLink>
-        </NavItem>
-        <NavItem>
-          <NavLink href="/users/sign_up">Sign up</NavLink>
-        </NavItem>
+        {
```

Comment:
Nitpick: you can reduce the number of React fragments by moving the condition inside the React fragments